### ISSUE TITLE
[Visual Requirements - Alert rules]: Keyboard focus indicator is not visible on "Seach by Data source" list options in high contrast mode.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66312]
### User Experience:
Users with low vision who depend on this functionality will not be able to identify the focus indicator on list items and they will not be able to identify the control hence face difficulty in performing actions.

### Pre-requisite:  
Windows - Applying High contrast theme. It doesn't matter what theme Grafana is using.

### Repro steps:
 Open #URL: https://play.grafana.org/  Sign in with valid credentials.  "Home" page appears. Navigate the page.  Navigate to the left navigation bar and to the "Alerting" button, select it.  It gets expanded and navigates the expanded sub menu items. Select the "Alert Rules" menu items. New page appears.  Navigate the tab which opens.  Navigate to "Search by data Sources" field and select it. Apply high contract theme. Verify the issue.

### Actual Result
Keyboard focus indicator is not visible on "Seach by Data source" list items in high contrast mode.  While navigating with down arrow key on expanded list options of "Seach by Data source" combo edit focus indicator is not visible in high contrast mode. Expected Result

 Keyboard focus indicator should be visible on "Seach by Data source" list items in high contrast mode, while navigating with down arrow key on expanded list option.
Note:  The same issue exists for "Choose Alertmanager" under silence tab on same screen. The same issue exists with the same type of controls throughout the application. same issue appears for explore-> query patterns.

### Area
Select Component (Saga UI Design System)

### Severity
`SEV2`

**Comment by joshhunt** [comment]
Related to high-contast theme issues in https://github.com/grafana/grafana/issues/66308
